sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,22 +1,17 @@
 # Release History
 
-## 1.8.0-beta.2 (Unreleased)
+## 1.8.0 (2023-02-03)
 
 ### Features Added
 
-- [[#32527]](https://github.com/Azure/azure-sdk-for-java/pull/32527) Added Azure Developer CLI Credential.
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+#### Features Generally Available from v1.8.0-beta1
```

Comment:
Just a suggestion, what you wrote is also fine.

```suggestion
#### From version 1.8.0-beta.1
```

Reply:
we've gotten some feedback on the Identity readmes from users that they want clarity about what is now GA so I want to use that language specifically.